### PR TITLE
Update proofr.Rmd

### PR DESCRIPTION
**Pull request by vortexing**
wee tweak of the vignette

### CODE REVIEW

File: vignettes/proofr.Rmd [hunk 1]
```diff
@@ -18,7 +18,7 @@ library(proofr)
 
 ## Authenticate with the PROOF API
 
-Run the function `proof_authenticate()`, which calls the PROOF API with your username and password, and returns an API token (an alphanumeric string).
+Run the function `proof_authenticate()`, which calls the PROOF API with your username and password, and returns an API token (an alphanumeric string). A new token is created when a new PROOF server is created, thus this API token will need to be reset.
```
Comment (dtenenba) [inline vignettes/proofr.Rmd]: This is not what happens. The token does not change. We can/should discuss whether it should, but currently it does not.

Comment (sckott) [reply vignettes/proofr.Rmd]: @vortexing can you undo this text change, or change the wording to explain that the same token is returned when calling `proof_authenticate`

**Comment by sckott** [comment]
@vortexing So maybe this isn't super clear (I'll make it more clear), but the original file that the vignette comes from is the file `proofr.Rmd.og`, and running `make vign_getting_started` creates `proofr.Rmd`. Okay if I port your changes on the `.Rmd` file into `.Rmd.og` file and remake `.Rmd`?

**Comment by tefirman** [comment]
@sckott @dtenenba -- I removed the token sentence talked about above, ported the changes over to "proofr.md.og", and remade "proofr.md" using `make vign_getting_started`. Does this look okay now?

**Review by sckott — APPROVED** [review]

**Comment by sckott** [comment]
Thanks @tefirman ! looks good

**Pull request merged**